demos/999-storage-import-vhd-blob/scripts/ImportVHD-FromBlobUri.ps1

```diff
@@ -0,0 +1,57 @@
+param(
```

Comment:
remove this file or remove the inline script in azuredeploy.json

Reply:
done